apps/portals/nf/src/config/synapseConfigs/organizations.ts

```diff
@@ -32,7 +32,8 @@ export const organizationDetailsPageConfig: DetailsPageProps = {
             sql: studiesSql,
             visibleColumnCount: 7,
             sqlOperator: ColumnSingleValueFilterOperator.LIKE,
-            cardConfiguration: { ...studyCardConfiguration, initialLimit: 2 },
+            initialLimit: 2,
+            cardConfiguration: studyCardConfiguration,
```

Comment:
`initialLimit` (which is pagination logic) was decoupled from `cardConfiguration` (which is "view" logic)